lighting/material/new.glsl

```diff
@@ -43,6 +43,8 @@ void materialNew(out Material _mat) {
     // Surface data
     _mat.position           = (SURFACE_POSITION).xyz;
     _mat.normal             = materialNormal();
+    _mat.sdf                = 0.0;
```

Comment:
Add `#if defined(RENDER_RAYMARCHING)` or `#if defined(SHADING_MODEL_SDF)` or similar. What ever seems better for you